default/python/README.md

```diff
@@ -28,15 +28,15 @@ We use [flake8-putty](https://pypi.python.org/pypi/flake8-putty)
 which include [flake8](https://pypi.python.org/pypi/flake8)
 which include [pycodestyle](https://pypi.python.org/pypi/pycodestyle) and other tools.
 
-`flake8-putty` is `flake8` plugin that allows one to disable rule for certain file, see `putty-ignore` section in `tox.ini`
+`flake8-putty` is `flake8` plugin that allows one to disable rule for certain file,
+see `putty-ignore` section in `tox.ini`. This allow to ignore certain
+warnings only for specified files, like bare excepts, that hard to fie,
+or special formatting in table like files.
```

Comment:
perhaps "files with tables"?
or "table-like files"?
We do, or did, have one of the latter for universe generation, though it's my recollection that's changed over now to being a python file with table-like entries, so probably "files with tables" is best.